includes/avatarMove.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Avatar Move</title>
+</head>
+<body>
+
+<svg id="svgOne" xmlns="http://www.w3.org/2000/svg" width="800" height="600" class="svgbox">
```

Comment:
does this need indenting?
does line spacing need to be tidied up?

Reply:
done